Makefile

```diff
@@ -301,10 +301,14 @@ vcpus:
 	bash -f integration/vcpus/default_vcpus_test.sh
 
 vfio:
-	bash -f functional/vfio/run.sh -s false -p qemu -m pc
-	bash -f functional/vfio/run.sh -s true -p qemu -m pc
-	bash -f functional/vfio/run.sh -s false -p qemu -m q35
-	bash -f functional/vfio/run.sh -s true -p qemu -m q35
+	bash -f functional/vfio/run.sh -s false -p qemu -m pc -i image
```

Comment:
What is the true/false flag here, trying to see what combinations we are testing here.
Stepping back, do we want to test so many combinations?

Reply:
sadly, yes, there are not enough resources to have a VFIO CI per feature (sandbox_cgroup_only, q35 or pc, image or initrd)

Reply:
@devimc so we want to keep these 8 tests for now?

Reply:
yes